collections/_get-started/customise-with-sass.html

```diff
@@ -52,10 +52,10 @@
 
 {% capture sgds-variable %}
 node_modules/
-  └── @govtechsg/sgds/sass/
-      └── bootstrap/
-      └── sgds-theme/
-          └── _sgds-variables.scss
+└── @govtechsg/sgds/sass/
```

Comment:
I thought the current folder structure (above in red) made more sense. how come bootstrap, sgds-theme folder and sgds-variable.scss file is at same level as @govtechsg/sgds

Reply:
Think i formatted it in vscode. Will revert

Reply:
resolved in commit [c84b9e6](https://github.com/GovTechSG/sgds/pull/372/commits/c84b9e6cc8c0cf24e86023430ae0b869aee2ea32)